client/my-sites/checkout/src/hooks/use-create-payment-methods/index.tsx

```diff
@@ -49,21 +50,28 @@ const debug = debugFactory( 'calypso:use-create-payment-methods' );
 
 export { useCreateExistingCards };
 
-export function useCreatePayPal( {
+export function useCreatePayPalExpress( {
 	labelText,
 	shouldShowTaxFields,
 }: {
 	labelText?: string | null;
 	shouldShowTaxFields?: boolean;
-} ): PaymentMethod {
+} ): PaymentMethod | null {
 	const store = useMemo( () => createPayPalStore(), [] );
+	const shouldUsePayPalPPCP = isEnabled( 'checkout/paypal-ppcp' );
```

Comment:
Note to self: PP Express and PPCP have different payment method IDs. If this feature flag is enabled and the endpoint does _not_ return the PPCP payment method as available but _does_ return PP Express, then we should probably show that instead.